docs/protocol/core/v3.0.rst

```diff
@@ -144,11 +144,11 @@ Questions that still need to be resolved
 We solicit feedback on the following area during the RFC period of this first
 draft.
 
- - Should core metadata and user attributes be stored together or separate documents? ([GH72](https://github.com/zarr-developers/zarr-specs/issues/72)) 
-   large metadata documents.
- - extensions and ``must_understand = True`` might be too restrictive. Work a
-   draft implementation with extensions and
-   see how far we can go. Possible list of extensions to implement:
+ - Should core metadata and user attributes be stored together or separate documents?
+   (See https://github.com/zarr-developers/zarr-specs/issues/72)
+ - extensions and ``must_understand = True`` might be too restrictive.
+   We propose to develop a draft implementation with extensions and
+   see how far we can go. A possible list of extensions to implement includes:
```

Comment:
I still feel like this language "see how far we can go" is not precise enough for a spec.